Feat/flask2

freesteph opened the pull request:
Upgrade to Flask 2 and fix the context request code for it.

Commit:
pip: remove a bunch of duplicate packages in the dev dependencies

Commit:
app: remove the lambda wsgi wrapper around the app

Commit:
chore: upgrade to flask v2

Ripped off https://github.com/cds-snc/notification-admin/commit/f7f1d6ab4137dc9b2499554a4aacb4b587f3dfe9

AntoineAugusti commented on app/templates/components/stepper.html:
```diff
@@ -0,0 +1,19 @@
+    {% if next_step %}
+      <p class="fr-stepper__details">
+        <span class="fr-text--bold">Étape suivante :</span> {{ next_step }}
+      </p>
```
Pas de traduction ici ?

freesteph replied:
ah si oui pardon faut que je prenne le réflexe

AntoineAugusti commented on Pipfile:
```diff
@@ -5,12 +5,11 @@ name = "pypi"
-flask = "==1.1.2"
-flask-wtf = "==0.14.3"
-flask-login = "==0.5.0"
+flask = "==2.2.2"
```
Super 😍

AntoineAugusti commented on Pipfile:
```diff
@@ -34,13 +33,12 @@ ua-parser = "==0.10.0"
-itsdangerous = "==0.24"
-notifications-utils = {ref = "47.0.0", git = "https://github.com/cds-snc/notifier-utils.git"}
+itsdangerous = "==2.0.1"
+notifications-utils = {git = "https://github.com/betagouv/notification-utils.git"}
```
💅

freesteph committed:
app: remove the custom commands

I don't think we'll be using it, and it breaks the app if it's
launched in a `pipenv` virtual env, which we'd like to leverage soon.

freesteph committed:
app: run commands via pipenv

This allows us to install things locally and not mess with system-wide
packages which comes in handy when debugging things.
